src/concurrent_map/cmap_class.cuh

```diff
@@ -249,7 +258,7 @@ class GpuSlabHash<KeyT, ValueT, SlabHashTypeT::ConcurrentMap> {
   std::string to_string();
   double computeLoadFactor(int flag);
 
-  void buildBulk(KeyT* d_key, ValueT* d_value, uint32_t num_keys);
+  void buildBulk(KeyT* d_key, ValueT* d_value, uint32_t num_keys, bool unique_keys = false);
```

Comment:
I think it's better if we separate the two functions. It's a little bit ambiguous to have this boolean argument to denote uniqueness or not.